### PR TITLE
Fix fe creator field

### PR DESCRIPTION
**Pull request by iainnash**
We need to get the creator address for contracts that are created with mint multicalls via the factory.

This fixes that concern by finding the new contract creation event and assigns the creator as the owner of the new contract.

### CODE REVIEW

**Comment by iainnash** [comment]
Deployment: https://app.goldsky.com/dashboard/subgraphs/zora-create-zora-testnet/1.16.0

File: src/ERC1155Mappings/templates/ZoraCreator1155ImplMappings.ts [hunk 1]
```diff
@@ -219,6 +219,51 @@ function _updateHolderTransfer(
...
+
+    // SetupNewContract(address indexed newContract, address indexed creator, address indexed defaultAdmin, ...);
+    if (
+      event.receipt!.logs[i].topics[0].equals(
+        Bytes.fromHexString(
+          "0xa45800684f65ae010ceb4385eceaed88dec7f6a6bcbe11f7ffd8bd24dd2653f4"
+        )
+      )
+    ) {
+      const newAddress = event
```
Comment (kulkarohan) [inline src/ERC1155Mappings/templates/ZoraCreator1155ImplMappings.ts:254]: yikes lol but nice trick

**Review by kulkarohan — APPROVED** [review]

File: src/ERC1155Mappings/templates/ZoraCreator1155ImplMappings.ts [hunk 2]
```diff
@@ -219,6 +219,51 @@ function _updateHolderTransfer(
...
+        .receipt!.logs[i].topics[3].toHexString()
+        .slice(96 + 2, 96 + 40 + 2);
```
Comment (kulkarohan) [inline src/ERC1155Mappings/templates/ZoraCreator1155ImplMappings.ts:256]: yikes lol but nice trick

**Review by IsabellaSmallcombe — APPROVED** [review]

File: src/ERC1155Mappings/templates/ZoraCreator1155ImplMappings.ts (hunk 2 above)
Comment (iainnash) [reply src/ERC1155Mappings/templates/ZoraCreator1155ImplMappings.ts]: yeah it's meh but it does work well.

**Pull request merged**